src/safeds/data/tabular/transformation/_discretizer.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+
+from sklearn.preprocessing import KBinsDiscretizer as sk_KBinsDiscretizer
+
+from safeds.data.tabular.containers import Table
+from safeds.data.tabular.transformation._table_transformer import TableTransformer
+from safeds.exceptions import TransformerNotFittedError, UnknownColumnNameError
+
+
+class Discretizer(TableTransformer):
+    """
+    The Discretizer transforms replaces a continuous variable by a variable that only has a finite amount of values.
+
+    Parameters
+    ----------
+    number_of_bins: float
+        The number of bins to be created.
+
+    Raises
+    ------
+    ValueError
+        If the given number_of_bins is less than 2.
+    """
+
+    def __init__(self, number_of_bins: float = 5):
+        self._column_names: list[str] | None = None
+        self._wrapped_transformer: sk_KBinsDiscretizer | None = None
+
+        if number_of_bins < 2:
+            raise ValueError("Parameter 'number_of_bins' must be >= 2.")
+        self.number_of_bins = number_of_bins
```

Comment:
```suggestion
        self._number_of_bins = number_of_bins
```
Please use an underscore before every private variable